Weatherman_using CSV.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: cp1252 -*-
+# created by Hassan Khalid
+# dated: 27-03-2017
+import glob
+import os
+import csv
+
+years_to_processed = ['1996','1997','1998','1999','2000','2001','2002','2003','2004','2005','2006','2007','2008','2009','2010','2011']
+files = []
+maximum_temperature = []
+dates = []
+monthly_max_temp = {}
+yearly_max_temp = {}
+
+def file_processing(file_name):
+    
+    minimum_temperature = []
+    maximum_humidity = []
+    minimum_humidity = []
+
+    dates[:] = []
+    maximum_temperature[:]=[]
+
+    max_temp = 0
+    min_temp = 0
+    max_humidity = 0
+    min_humidity = 0
+ 
+
+    with open(file_name, 'r') as f:
+        reader = csv.reader(f,delimiter=',')

```

Comment:
This is simple csv.reader. I specifically mentioned [DictReader](https://docs.python.org/3/library/csv.html#csv.DictReader)